hasura/Dockerfile

```diff
@@ -5,21 +5,24 @@ RUN chmod +x /wait
 
 ## Default setup
 
+ARG BACKEND_HOST
+ARG AUTH_HOOK_PATH=auth-webhook
+ARG ACTION_BASE_PATH=actions
+
+ENV HASURA_GRAPHQL_DEV_MODE false
 ENV HASURA_GRAPHQL_ENABLE_TELEMETRY false
 ENV HASURA_GRAPHQL_ENABLED_LOG_TYPES startup, http-log, webhook-log, websocket-log, query-log
+ENV HASURA_GRAPHQL_AUTH_HOOK http://$BACKEND_HOST/$AUTH_HOOK_PATH
```

Comment:
@pakokrew The reason we have to do it like this is Render not allowing for string interpolation when defining ENV vars based on other services:
![image](https://user-images.githubusercontent.com/7143583/86569997-7cb62700-bf2c-11ea-8be8-cf35f29448e4.png)